lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+    description:
+      - Path of the authentication file. Default is ``${XDG_RUNTIME_DIR}/containers/auth.json``
+        (Not available for remote commands) You can also override the default path of the authentication file
+        by setting the ``REGISTRY_AUTH_FILE`` environment variable. ``export REGISTRY_AUTH_FILE=path``
+    type: str
+  blkio_weight:
+    description:
+      - Block IO weight (relative weight) accepts a weight value between 10 and 1000
+    type: int
+  blkio_weight_device:
+    description:
+      - Block IO weight (relative device weight, format DEVICE_NAME[:]WEIGHT).
+    type: str
+  cap_add:
+    description:
+      - List of capabilities to add to the container.
+    type: list
+  cap_drop:
+    description:
+      - List of capabilities to drop from the container.
+    type: list
+  cgroup_parent:
+    description:
+      - Path to cgroups under which the cgroup for the container will be created.
+        If the path is not absolute, the path is considered to be relative to
+        the cgroups path of the init process. Cgroups will be created if they do not already exist.
+    type: str
+  cidfile:
+    description:
+      - Write the container ID to the file
+    type: str
+  cmd_args:
+    description:
+      - Any additionl command options you want to pass to podman command, cmd_args - ['--other-param', 'value']
+    type: list
+  conmon_pidfile:
+    description:
+      - Write the pid of the conmon process to a file.
+        conmon runs in a separate process than Podman,
+        so this is necessary when using systemd to restart Podman containers.
+    type: str
+  command:
+    description:
+      - Override command of container
+    type: raw
+  cpu_period:
+    description:
+      - Limit the CPU real-time period in microseconds
+    type: int
+  cpu_rt_runtime:
+    description:
+      - Limit the CPU real-time runtime in microseconds
+    type: int
+  cpu_shares:
+    description:
+      - CPU shares (relative weight)
+    type: int
+  cpus:
+    description:
+      - Number of CPUs. The default is 0.0 which means no limit.
+    type: str
+  cpuset_cpusr:
+    description:
+      - CPUs in which to allow execution (0-3, 0,1)
+    type: str
+  cpuset_mems:
+    description:
+      - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
+    type: str
+  detach:
+    description:
+      - Run container in detach mode
+    type: bool
+    default: True
+  detach_keys:
+    description:
+      - Override the key sequence for detaching a container. Format is a single character or ctrl-value
+    type: str
+  device:
+    description:
+      - Add a host device to the container.
+        The format is <device-on-host>[:<device-on-container>][:<permissions>]
+        (e.g. device /dev/sdc:/dev/xvdc:rwm)
+    type: str
+  device_read_bps:
+    description:
+      - Limit read rate (bytes per second) from a device (e.g. device-read-bps /dev/sda:1mb)
+    type: str
```

Comment:
Also here: shouldn't that be a list? Or better a dict, or even a list of dicts (similar to `docker_container`)?

Reply:
(Same for the related options.)